### ISSUE TITLE
dm2e:incipit & dm2e:explicit should be repeatable

### ISSUE THREAD

**Issue by d0rg0ld** [DM2E/dm2e-mappings#43]
Just came across such a case:

```
  <dm2e:incipit>Prospero prudente constate felyce ... [Prolog]</dm2e:incipit>
  <dm2e:incipit>EntonÃ§es se apareja la cosa ...</dm2e:incipit>
  <dm2e:explicit>... Fable la obra que el dezir se Ã§ierra. [Prolog]</dm2e:explicit>
  <dm2e:explicit>... ny demandarles cosas non acostumbradas.</dm2e:explicit>
```

**Comment by d0rg0ld** [comment]
for now i will probably just merge all entries

**Comment by kba** [comment]
Is not an error in 1.1_Rev1.5-DRAFT anymore.

**Comment by edroege** [comment]
We have thought that opening and ending words of a manuscript would only occur once. Could you explain the case @d0rg0ld  ? Are these different languages or different editions of a manuscript? Shouldn't they be two different CHOs then?

**Comment by d0rg0ld** [comment]
As you can see in the example above, I encounter it in the metadata. It looks like the same language to me and the tag [prolog] suggests that the first line of each pair seems to be the beginning of the incipit/explicit and the other its end, respectively. 

According to http://www.hbz-nrw.de/dokumentencenter/produkte/verbunddatenbank/aktuell/plausi/TIT.pdf MAB Field 661 ( With indicator a being the Incipit) can be repeated 25 times.

Therefore it should be repeatable ...

**Comment by edroege** [comment]
Okay, fixed in model revision 1.5

**Comment by d0rg0ld** [comment]
thank you!
